lib/cryptoexchange/exchanges/coinzest/services/market.rb

```diff
@@ -0,0 +1,53 @@
+module Cryptoexchange::Exchanges
+  module Coinzest
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+        
+        def fetch
+          ctx = OpenSSL::SSL::SSLContext.new
+          ctx.verify_mode = OpenSSL::SSL::VERIFY_NONE
+          result = HTTP.get(ticker_url, ssl_context: ctx)
+          output = JSON.parse(result)
+          adapt_all(output)
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::Coinzest::Market::API_URL}/market_summary"
+        end
+
+        def adapt_all(output)
+          output['result'].map do |pair|
+            base, target = pair['MarketName'].split('-')
```

Comment:
`base` and `target` inverted. If you look at the exchange page. https://www.coinzest.co.kr/app/cxweb/it/IT4001.jsp
BTC is base
KRW is target

and the `BaseVolume` is actually target's volume, so you have to derive the volume using the price.